### DISCUSSION TITLE
Coverage plot for multiome data

### DISCUSSION THREAD

**Discussion by shamimaTU** [satijalab/seurat#7068] [Q&A] [answered]
I would like to get a plot like this one , but so far I have not managed to sort it out. I am able to create a plot for a single gene but not for adding many genes in the same plot. Thanks in advance!
<img width="1059" alt="coverage" src="https://user-images.githubusercontent.com/92605695/227363537-8e137f3b-cc79-437b-840c-e1b991bbe6d2.png">

**Comment by timoast** [comment, marked as answer]
https://github.com/stuart-lab/signac/issues/362
